integreat_cms/cms/forms/users/user_form.py

```diff
@@ -111,6 +112,12 @@ def __init__(self, **kwargs):
             self.fields["is_staff"].label = _("Integreat team member")
         self.fields["email"].required = True
 
+        # Check if passwordless authentication is possible for the user
+        # if self.instance.totp_key is None and not self.instance.mfa_keys.exists():
```

Comment:
@timoludwig Hiermit kann ich zumindest client-seitig auch nochmal eine extra Validierung hinzufügen, aber wenn ich das auskommentiere, failen eine Reihe von Tests, was ich mich nicht ganz erklären kann :thinking:

Reply:
This is because the region user form inherits from the user form, but with different fields... meaning that the same `__init__()` method is called, but accessing a non-existing field results in a key error. So I'd suggest to check whether the field really exists before making it disabled.